src/hooks/useEagerConnect.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react'
+import { useWeb3React } from '@web3-react/core'
+import { connectorsByName } from 'utils/web3React'
+
+const { injected } = connectorsByName
+
+const useEagerConnect = () => {
+  const { activate } = useWeb3React()
+
+  useEffect(() => {
+    injected.isAuthorized().then((isAuthorized) => {
+      const hasSignedIn = window.localStorage.getItem('accountStatus')
```

Comment:
We should consider using cookies if we plan on keeping the exchange on a different domain and we want to share the login logic.